lsp-mode.el

```diff
@@ -5499,18 +5500,26 @@ A reference is highlighted only if it is visible in a window."
     (19 . "Object")
     (20 . "Key")
     (21 . "Null")
-    (22 . "Enum Member")
+    (22 . "Enumerator")
```

Comment:
Since I'm not native speaker, I'm not sure about this change. But I think the meaning of `enumerator` and `enum member` is different.
The [LSP specs](https://github.com/microsoft/language-server-protocol/blob/gh-pages/_specifications/specification-3-16.md) is using `EnumMember` so we should probably stick with that.

Reply:
This probably depends on the programming language and context, but in C++ and "Enumerator" is an "Enum member": https://en.cppreference.com/w/cpp/language/enum, Unscoped declaration (1): `enum` [name] { *enumerator* = [...] }

Reply:
If it is language-specific we should stick with the LSP spec term.

Reply:
I thought it was a standard term, but the Java spec doesn't use it, preferring "Enum Constant". I'm reverting.